controllers/rooms.controller.js

```diff
@@ -1,12 +1,51 @@
 import fs from "fs";
+import uuid from "uuid/v4";
+
+const roomFilePath = "./file/matrix.room.web.json"
 
 const fetchFromFile = () => {
-  const roomsData = fs.readFileSync("./file/default.room.web.json");
+  const roomFileExists = fs.existsSync(roomFilePath)
+  if (!roomFileExists) {
+    createRoomFileSync()
```

Comment:
Some users can use file for rooms source, and run the app at heroku. Maybe it will generate a new rooms file every deploy, no?

Reply:
Yes, but we can recommend for these users to configure an environment variable with the static room list. What do you think about this?

Reply:
Only if the file does not exists, that means the user haven't set the file as the source, relying in the default one, but we can't have a default file because ever matrix user would enter in the same meeting.
The environment variable should be the recommended way to configure the environment, but if the user haven't set it, it is better to use a reasonable default, in this case a random generated room IDs is not so bad. Are you thinking in a better alternative?